### DISCUSSION TITLE
Generate pip package for last release!

### DISCUSSION THREAD

**Discussion by perseoGI** [Unleash/unleash-client-python#261] [General]
Please, could you generate a new release for v5.7.0.

Thare has been some improvements on this release and the last CI failed to publish.

**Comment by ivanklee86** [comment]
Whoops, thanks for letting us know!  Will fix soon.  😄

**Comment by ivanklee86** [comment]
This is fixed and v5.7.0 is on Pypi!

**Reply by perseoGI** [reply]
Niceeee!!! Thank you very much!

**Reply by ivanklee86** [reply]
Whoops, there's a separate issue in `v5.7.0` .  Working on a fix, `v5.7.1` will be out shortly.
